### PR TITLE
[wxWidgets] Fix build with clang-cl

### PR DESCRIPTION
**Pull request by Neumann-A**
needs to include stdint.h for uinptr and stuff instead of stddef.h

### CODE REVIEW

File: versions/baseline.json [hunk 1]
```diff
@@ -6060,6 +6060,10 @@
       "baseline": "2021-02-21",
       "port-version": 0
     },
+    "ptc-print": {
+      "baseline": "1.4.0",
+      "port-version": 0
+    },
```
Comment (Cheney-W) [inline versions/baseline.json:6066]: This part of the modification should not exist. It seems that your local `baseline.json` is not the latest.

**Review by Cheney-W — CHANGES_REQUESTED** [review]

File: versions/baseline.json (hunk 1 above)
Comment (dg0yt) [reply versions/baseline.json:6066]: This part is due to `baseline.json` not being properly formatted, cf. https://github.com/microsoft/vcpkg/commit/b10c0ba13ed067a59447339ee8143fd875934a84#r95116208 
Every user of `vcpkg x-add-version` will get this change. You can fix it in a separate PR. But identical changes shouldn't cause merge conflicts, so there is no reason to ask for changes.

Comment (Cheney-W) [reply versions/baseline.json:6066]: That issue has been fixed by https://github.com/microsoft/vcpkg/commit/a6280656930661139383fe39527b9ea5de160302.

Comment (dg0yt) [reply versions/baseline.json:6066]: ... which picked that change in the same way. 
If there is no merge conflict, there is no need for contributor action.

**Review by Cheney-W — APPROVED** [review]

File: versions/baseline.json (hunk 1 above)
Comment (Cheney-W) [reply versions/baseline.json:6066]: OK, I got it. Thank you!

**Pull request merged**